src/bentoml/testing/server.py

```diff
@@ -151,24 +151,26 @@ def containerize(
     cleanup: bool = True,
     use_grpc: bool = False,
     backend: str = "docker",
+    **attrs: t.Any,
 ) -> t.Generator[str, None, None]:
     """
     Build the docker image from a saved bento, yield the docker image tag
     """
-    from bentoml import bentos
+    from bentoml import container
 
     bento_tag = Tag.from_taglike(bento_tag)
     if image_tag is None:
-        image_tag = bento_tag.name
+        image_tag = str(bento_tag)
     try:
         print(f"Building bento server container: {bento_tag}")
-        bentos.containerize(
+        container.build(
             str(bento_tag),
-            docker_image_tag=(image_tag,),
+            backend=backend,
+            image_tag=(image_tag,),
             progress="plain",
             features=["grpc", "grpc-reflection"] if use_grpc else None,
-            labels={"testing": True, "module": __name__},
-            backend=backend,
+            label={"testing": True, "module": __name__},
```

Comment:
Wait, why did we change this in the API?

Reply:
because it is actually `--label` from docker, not `--labels`.

`bentoml.bentos.containerize` provides backward compatibility from `labels` to `label`.